### DISCUSSION TITLE
Remult is a good framework. But how can I use it without frontend code.

### DISCUSSION THREAD

**Discussion by zhixXiang** [noam-honig/radweb#143] [Q&A] [answered]
If I only wanted to use Remult to create a restful api project without any frontend code, any suggestion about this?

**Comment by noam-honig** [comment, marked as answer]
You can add remult to any backend project, just follow the steps at the [Add remult to your app](https://remult.dev/docs/add-remult-to-your-app.html#installation) article.

Here's an express remult started project you can use as a starting point or an example
https://github.com/noam-honig/express-starter.

Let me know if that helps

**Reply by zhixXiang** [reply]
Thank you, this is what i need!
